src/Microsoft.DotNet.Build.Tasks.net45/project.json

```diff
@@ -1,11 +1,16 @@
 {
-  "dependencies": {
-    "Newtonsoft.Json": "7.0.1",
-    "NuGet.Versioning": "3.4.0-beta-488",
-    "System.Collections.Immutable": "1.1.37",
-    "System.Reflection.Metadata": "1.1.0"
-  },
-  "frameworks": {
-    "net45": { }
-  }
-}
\ No newline at end of file
+    "dependencies": {
+        "Newtonsoft.Json": "7.0.1",
+        "NuGet.Versioning": "3.4.0-beta-488",
+        "System.Collections.Immutable": "1.1.37",
+        "System.Reflection.Metadata": "1.1.0",
+        "Microsoft.NETCore.Platforms": "1.0.1-rc2-23907",
+        "System.Runtime.InteropServices.RuntimeInformation": "4.0.0-rc2-23907"
+    },
+    "frameworks": {
+        "net46": { }
```

Comment:
@ericstj looks like you did the same trick in the packaging project.json. Is it the RID inference you are trying to workaround here or do we need to actually fix something in our packages.